src/functions.js

```diff
@@ -324,10 +324,48 @@ var _ = Mavo.Functions = {
 
 	len: text => str(text).length,
 
+	/**
+     * Search if a group, collection, or primitive contains needle
+	 * @returns Boolean if a haystack of object or primitive is passed
+	 * @returns Array of booleans if a haystack of array is passed
+     */
+    contains: (haystack, needle) => {
+		var ret = Mavo.Script.binaryOperation(haystack, needle, {
+			scalar: (haystack, needle) => {
+				var type = $.type(haystack);
+				if (type === "object") {
+					for (var property in haystack) {
+						ret = _.contains(haystack[property], needle);
+						if (Array.isArray(ret)) {
+							ret = Mavo.Functions.or(ret);
+						}
+						if (ret) {
+							return true;
+						}
+					}
+				}
+				else {
+					return _.search(haystack, needle) >= 0;
+				}
+				return ret;
+			},
+			identity: null
+		});
+		// if result is an empty array, return false
+		if (ret.length === 0) {
+			return false;
+		}
+		return ret;
```

Comment:
Nitpick: Blank lines between blocks of code (especially before control flow statements if they are not the only statements inside their parent) can improve readability. Check the rest of the code for examples.

This is up to you though, it won't hold up the PR.